src/FunctionApps/python/tests/IntakePipeline/extensions_test.py

```diff
@@ -0,0 +1,87 @@
+import pytest
+from unittest import mock
+
+import json
+import pathlib
+
+from phdi_transforms.geo import GeocodeResult
+
+from IntakePipeline.transform import find_patient_resources, transform_bundle
+
+
+@pytest.fixture()
+def bundle():
+    return json.load(
+        open(pathlib.Path(__file__).parent / "assets" / "patient_bundle.json")
+    )
+
+
+@mock.patch("IntakePipeline.transform.geocode")
+def test_add_extensions_to_patient(patched_geocode, bundle):
+    patched_geocode.return_value = GeocodeResult(
+        key="123 Fake St New York, NY 10001",
+        address=["123 FAKE ST", "UNIT 3"],
+        city="NEW YORK",
+        state="NY",
+        zipcode="10001",
+        fips="36061",
+        lat=45.123,
+        lng=-70.234,
+        county_fips="dunno",
+        county_name="no idea",
+        precision="close-ish",
+    )
+    patient = find_patient_resources(bundle)[0]
+
+    expected = {
```

Comment:
It seems like it may be a little brittle to use the entire bundle here - what about testing just the extensions?

Reply:
Sure we can do that instead